docs/source/pages/Algorithm.rst

```diff
@@ -0,0 +1,36 @@
+Algorithm
+=========================================================
+
+The membrane curvature MDANAlysis module calculates mean and gaussian curvature 
+from surfaces derived by a group of reference. By default, the group of reference 
+implemented are lipid headgroups.
+
+The algorithm locates all the atoms in the group of reference and map each element 
+of the group to a grid of `m x n` dimensions. The dimensions of the grid are 
+determined by the length of the simulation box in the first frame of the 
+trajectory.
+
+The grid is then divided in `n x m` elements, with unit cells of width 2nm (default 
+value). In this way, for every atom in the group of reference an indexed cell in 
+the grid is assigned. i.e. `(x, y) ↦ [i, j]`. 
+
+|grid|
+
+Once the grid is populated with the atoms in the group of reference, the respective 
+`z` coordinate is stored in element in an individual array of each `[i, j]`. This 
```

Comment:
Could you specify that either every z coordinate for every atom in that cell is stored, or the mean coordinate?

Reply:
I changed this paragraph a little bit.